### PR TITLE
Better README

### PR DESCRIPTION
**Pull request by hifinerd**
Pretty self-explanatory. This adds instructions on how to build MagmaWM.

### CODE REVIEW

**Comment by HackedOS** [comment]
Uhh the depends are not resolved by cargo, you also need to add instructions about setting up seatd properly

**Comment by hifinerd** [comment]
> you also need to add instructions about setting up seatd properly
Is there any particular reference for that I should take a look at?

**Comment by HackedOS** [comment]
You can look at the ci workflow file for the dependencies list (just translate them to dofferent distros)

**Comment by HackedOS** [comment]
Please revert back to stable

File: README.md [hunk 1]
```diff
@@ -1,17 +1,57 @@
-# MagmaWM: Magma Window Manager (and Compositor)
-## Work In Progress
 <p align="center">
+ <h1 align="center">MagmaWM</h1>
+  <p align="center">
     <img src="https://cdn.discordapp.com/attachments/1090909610828898304/1101557907453120545/Magma_Concept_3__2.png" style="width: 15%;" alt="wmlogo"></img>
```
Comment (electron271) [inline README.md:4]: do not use discord's CDNs to be safe, use your own website

Comment (electron271) [reply README.md:4]: also put the logo above the name

File: README.md [hunk 2]
```diff
@@ -1,17 +1,57 @@
...
+ <h3 align="center">a versatile and customizable Wayland Compositor</h3>
 </p>
+</p>
+  <p align="center">
+    <img src="https://img.shields.io/github/languages/top/magmawm/magmawm?style=for-the-badge"/>
+    <img src="https://img.shields.io/github/commit-activity/m/magmawm/magmawm?style=for-the-badge"/>
+    <img src="https://img.shields.io/github/license/magmawm/magmawm?style=for-the-badge"/>
```
Comment (electron271) [inline README.md:11]: license is not really needed as a badge

File: README.md [hunk 3]
```diff
@@ -1,17 +1,57 @@
...
+
+When you have finished making all of your changes, commit them using [Git](https://git-scm.com/) and create a pull request.
+
+## Dependencies
+Depedencies needed to compile MagmaWM
+### Arch
+(TO BE ADDED BY HIFI)
 
 ## Install
 **MagmaWM** is still under devlopment and can't be installed yet.
```
Comment (electron271) [inline README.md:56]: just at least put "If you really want to, run `cargo install --path .`"

File: README.md [hunk 4]
```diff
@@ -1,17 +1,57 @@
...
+To contribute your own code, start by forking this project and installing the necessary dependencies listed below.
 
-Currently Planned Featured to be Built-in by Default:
-* RON Configuration
-* Dynamic Tiling and Floating Windows
-* Keyboard and Monitor Managament
-* Screencopy Protocols for Screensharing/Screenshots
-* Lots of eye candy like blur, animations etc
+After installing the dependencies, you can build the project with `cargo build --release`.
+
```
Comment (electron271) [inline README.md:38]: also suggest `cargo run --release`

**Review by electron271 — CHANGES_REQUESTED** [review]

**Comment by HackedOS** [comment]
Add a section on how to report bugs that is, reading and sending logs etc, and add `cargo fmt` and `cargo clippy` to the contributing part

**Comment by HackedOS** [comment]
Add dependency install commands for few pther distros mainly arch

**Comment by hifinerd** [comment]
PR has been updated with the requested changes.

**Review by HackedOS — CHANGES_REQUESTED** [review]
Popups and RON config are added, add a section about cargo fmt and cargo clippy to contributing pls

**Comment by HackedOS** [comment]
keyboard and monitor config is also completed

**Comment by Insert5StarName** [comment]
> keyboard and monitor config is also completed

Full Keyboard Config?
layout etc?
Custom Remapping Caps to ctrl as an example?

**Comment by Insert5StarName** [comment]
> > > keyboard and monitor config is also completed
> > 
> > 
> > Full Keyboard Config? layout etc? Custom Remapping Caps to ctrl as an example?
> 
> yes, @egreg10us has already tested aswell

Monitors are done too?
Binding Workspaces to Monitor?
etc?

**Comment by HackedOS** [comment]
> > > > keyboard and monitor config is also completed
> > > 
> > > 
> > > Full Keyboard Config? layout etc? Custom Remapping Caps to ctrl as an example?
> > 
> > 
> > yes, @egreg10us has already tested aswell
> 
> Monitors are done too? Binding Workspaces to Monitor? etc?

only setting resolution and refresh rate done rn

**Pull request merged**